src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discount1 = 20;
+  const discount2 = 50;
```

Comment:
Please change variable names to more logical
![2023-02-10_15h00_11](https://user-images.githubusercontent.com/61904995/218098551-756d4e1f-eae3-4fd2-989f-8d5634cdb5bf.png)